RestartWindow.qml

```diff
@@ -433,7 +433,7 @@ Popup {
                         Text {
                             id: over_voltage_text
 
-                            property bool over_voltage_fault: (backEnd.pack_voltage > 108) && backEnd.restart_enable
+                            property bool over_voltage_fault: (backEnd.pack_voltage > 113.15) && backEnd.restart_enable
```

Comment:
maybe consider making this come from the sc1-data-format rather than being hardcoded

Reply:
#57